### DISCUSSION TITLE
mmc.exe fails to open on windows 10

### DISCUSSION THREAD

**Discussion by dennis777** [utmapp/UTM#6602] [Windows]
**Describe the issue**  
mmc does not open on windows 10. I get a popup saying This app has been blocked for your protection. When I go to windows security settings to disable app control, I cant click into the submenu to do so because I get a message saying this app cant open.
Is there anyway to bypass it to open mmc or some other way to disable windows security?

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: 15.0b
* Mac Chip: M1 Max 

**Debug log**  
[debug.log](https://github.com/user-attachments/files/16383781/debug.log)

**Upload VM**  
[config.plist.zip](https://github.com/user-attachments/files/16383786/config.plist.zip)

<img width="486" alt="Screenshot 2024-07-25 at 14 03 56" src="https://github.com/user-attachments/assets/16034124-cc7f-4158-92f5-d5d8670a6bea">
<img width="980" alt="image" src="https://github.com/user-attachments/assets/afdf9a83-2eec-49b1-bac2-5aeba7fd369f">

**Comment by romlu2024** [comment]
This is a pure Windows issue, not related to UTM.

Maybe this can help: https://www.auslogics.com/en/articles/fix-an-administrator-has-blocked-you-from-running-this-app/. (Check Fix 9)

Best regards

**Comment by dennis777** [comment]
I cannot resolve using fix 9 since I can't even open the registry editor. 
<img width="467" alt="image" src="https://github.com/user-attachments/assets/9b9a7ca6-0f7d-4f37-839b-559b7c1ebcbc">
